R/hooks.R

```diff
@@ -14,4 +14,9 @@
     packageStartupMessage("Assigning sp_evolution_status to 2. See sp::get_evolution_status()")
     packageStartupMessage("This is required until the 'sp' package deprecates 'rgdal'")
   }
+  if (requireNamespace("sp", quietly = TRUE)) {
```

Comment:
this statement is already present in the 4 lines above it

Reply:
I noticed in the readme you still ask users to set the sp_evolution_status manually as well, is that still necessary or does this statement take care of it already?